proto/v1/vulnerability.proto

```diff
@@ -174,6 +174,17 @@ message VulnerabilityOccurrence {
   // severity.
   float cvss_score = 3;
 
+  // The CVSS v3 score for this vulnerability.
+  message CVSSV3 {
+    // The base score for for this vulnerability according to cvss v3.
+    float base_score = 1;
+    // The severity rating assigned to this vulnerability by vulnerability provider.
+    Severity severity = 2;
```

Comment:
I thought we wanted to expose only base_score for now? Did the understanding change?

Reply:
yes there was a thread with Shmuel and Brian and we agreed to add in the severity as well.

Reply:
Commented on the doc as I don't see the mention of it anywhere. SGTM with the overall approach to record it.